browser-config.js

```diff
@@ -1,6 +1,21 @@
+const { sortMemberRules } = require('./sort-member-config');
+
 module.exports = {
-  "extends": "./index.js",
-  "env": {
-    "browser": true
-  }
+	"extends": "./index.js",
+	"parser": "babel-eslint",
+	"env": {
+		"browser": true
+	},
+	"plugins": [
+		"html",
+		"sort-class-members"
+	],
+	"globals": {
+		"D2L": false,
+		"Promise": false
+	},
+	"rules": {
+    "no-console": ["error", { "allow": ["warn", "error"] }],
```

Comment:
Are there other rules that were switched that we didn't expect?  Seems like a bunch might have changed, and console is the only one we noticed so far.